lib/aprb.ex

```diff
@@ -8,12 +8,12 @@ defmodule Aprb do
 
     children = [
       supervisor(Aprb.Repo, []),
-      worker(Aprb.Service.AmqEventService, ["conversations"], id: :conversations),
-      worker(Aprb.Service.AmqEventService, ["inquiries"], id: :amq_inquiries),
-      worker(Aprb.Service.AmqEventService, ["radiation.messages"], id: :radiation_messages),
-      worker(Aprb.Service.AmqEventService, ["subscriptions"], id: :subscriptions),
-      worker(Aprb.Service.AmqEventService, ["auctions"], id: :auctions),
-      worker(Aprb.Service.AmqEventService, ["purchases"], id: :purchases),
+      worker(Aprb.Service.AmqEventService, [%{topic: "conversations"}], id: :conversations),
```

Comment:
I _think_ first argument needs to be an array to make it keyword list, take a look at their example here https://hexdocs.pm/elixir/Supervisor.html, we could also follow their example where we could have a mandatory `topic` and have `routing_key` an optional thing in opts. What do you think?